docs/setup/jetson.md

```diff
@@ -38,33 +38,199 @@
 ## Before you start
 !!! attention
     **These directions are written for someone with experience with embedded Linux and basic embedded computers.**
+
 It is highly recommended to read through the getting started document for your NVIDIA® Jetson™ developer kit before beginning:
 
+* [Jetson Orin Nano Developer Kit Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-orin-nano-devkit)
 * [Getting Started With Jetson Xavier NX Developer Kit](https://developer.nvidia.com/embedded/learn/get-started-jetson-xavier-nx-devkit)
 * [Getting Started with Jetson Nano Developer Kit](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit)
 * [Getting Started with Jetson Nano 2GB Developer Kit](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-2gb-devkit)
 
 
-## Step-by-step
+## Setup Overview
+
+![](./data/jetson_setup_flow_overview.png)
+
+## ROS 2 Setup on Jetson
+
+### Use pre-built ROS 2 Docker container
+
+The recommended way to run ROS2 on Jetson is to use pre-built Docker container images.
```

Comment:
Please fix many instances of "ROS2" (see https://twitter.com/OpenRoboticsOrg/status/1628514177009713155)

Reply:
[f5c1c96](https://github.com/iRobotEducation/create3_docs/pull/385/commits/f5c1c96d6181e7571c3a4f26ec0272fce4742728)